routers/api/v1/shared/action_list_runs_test.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2025 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package shared
+
+import (
+	"net/url"
+	"testing"
+	"time"
+
+	actions_model "code.gitea.io/gitea/models/actions"
+	"code.gitea.io/gitea/models/unittest"
+	"code.gitea.io/gitea/services/context"
+	"code.gitea.io/gitea/services/contexttest"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	unittest.MainTest(m)
+}
+
+// setFormValue is a helper function to set form values in test context
+func setFormValue(ctx *context.APIContext, key, value string) {
```

Comment:
IIRC our framework fills the request query&form if you call `contexttest.MockAPIContext(t, "user2/repo1?key=value")`

Reply:
Addressed, see https://github.com/go-gitea/gitea/pull/34894#issuecomment-3016900408 and https://github.com/go-gitea/gitea/pull/34894/commits/8b7ed3af421ade2adabc9896bca8d4871029b2d0